src/components/Button.jsx

```diff
@@ -1,3 +1,18 @@
-const Button = () => {};
+import Link from "next/link";
 
+const Button = ({ color, text }) => {
```

Comment:
you can remove the color parameter from here, the button will be the same color always